requirements.txt

```diff
@@ -43,7 +43,7 @@ python-rapidjson==1.4
 redis==2.10.6
 redis-py-cluster==1.3.5
 sentry-arroyo==0.0.4
-sentry-relay==0.8.7
+sentry-relay==0.8.9
```

Comment:
The new release includes `aarch64` wheels, thus, removing the need to build from source and installing relay build tools & libraries.